config/train_parametrs.yaml

```diff
@@ -0,0 +1,9 @@
+random_seed: 13
+cudn_determenistic: True 
+num_classes: 15
+data_zip_path: 'archive_full.zip'
+data_path: 'data/'
```

Comment:
what's the difference between data_zip_path and data_path? Why the data_path is neccessary if you are unpacking from archive. Maybe the name `data_path` should be `data_save_path` or another?